examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/WildcardFragment.kt

```diff
@@ -64,6 +71,24 @@ class WildcardFragment : Fragment() {
     }
   }
 
+  private val writeAttributeCallback = object : WriteAttributesCallback {
+    override fun onError(attributePath: ChipAttributePath?, ex: Exception?) {
+      if (attributePath != null)
+      {
+        Log.e(TAG, "Report error for $attributePath: $ex")
```

Comment:
Might be worth writing this error even if the attribute path is null

Reply:
I Modified. Thanks.